backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -1052,9 +1075,11 @@ public Project logArtifacts(String projectId, List<Artifact> newArtifacts)
       }
 
       Map<Long, Workspace> cacheWorkspaceMap = new HashMap<>();
+      Map<String, GetResourcesResponseItem> getResourcesMap = new HashMap<>();
```

Comment:
For example, this is a map without the project id as key.

Reply:
got it. Fixed